src/main/java/nextstep/subway/line/domain/Color.java

```diff
@@ -0,0 +1,53 @@
+package nextstep.subway.line.domain;
+
+import java.util.Objects;
+import javax.persistence.Column;
+import javax.persistence.Embeddable;
+import nextstep.subway.common.constant.ErrorCode;
+import nextstep.subway.utils.StringUtils;
+
+@Embeddable
+public class Color {
+
+    @Column(nullable = false)
+    private String color;
+
+    protected Color() {
```

Comment:
```suggestion
    protected Color() {}
```
이 부분은 중요한 것은 아니지만, [컨벤션](https://naver.github.io/hackday-conventions-java/#permit-concise-empty-block)에 따르면 이렇게 수정해줄 수도 있을 것 같아요...!

Reply:
해당 컨벤션 참고하여서 수정해보겠습니다 ㅎㅎ 좋은 자료 감사합니다 :)